projects/emissary-ingress/emissary/docker/linux/emissary-image/Dockerfile

```diff
@@ -0,0 +1,65 @@
+ARG BASE_IMAGE
+
+FROM public.ecr.aws/k1e6s8o8/eks-distro-minimal-base-python-builder:3.9-latest.2 as emissary-builder
+
+ENV PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/go/bin:/buildroot/bin
+
+RUN install_rpm libcap htop && \
+    install_binary /usr/bin/curl /usr/bin/chgrp /usr/bin/chmod && \
+    cleanup "emissary"
+
+WORKDIR /buildroot
+
+ADD emissary/python python
+
+RUN time pip3 install --no-deps -e python
+
+RUN pip3 install pip-tools==6.3.1
+RUN pip3 install PyYAML==5.4.1
+RUN pip3 install orjson==3.6.6
+
+RUN mkdir -p /ambassador/sidecars && \
+    ln -s /buildroot/python/post_update.py /ambassador/post_update.py && \
+    ln -s /buildroot/python/watch_hook.py /ambassador/watch_hook.py && \
+    ln -s /buildroot/python/kubewatch.py /ambassador/kubewatch.py
+
+COPY _output/bin/emissary/linux-amd64 /buildroot/bin
+
+RUN ln -s /buildroot/bin/busyambassador /buildroot/bin/kubestatus && \
+    ln -s /buildroot/bin/busyambassador /buildroot/bin/watt && \
+    ln -s /buildroot/bin/busyambassador /buildroot/bin/agent && \
+    ln -s /buildroot/bin/busyambassador /buildroot/bin/apiext && \
+    ln -s /buildroot/bin/* /usr/local/bin/
+
+FROM public.ecr.aws/appmesh/aws-appmesh-envoy:v1.22.2.0-prod as envoy
+FROM public.ecr.aws/k1e6s8o8/eks-distro-minimal-base-python:3.9-latest.2
+
+ARG TARGETARCH
+ARG TARGETOS
+
+ADD emissary/manifests/emissary/emissary-crds.yaml.in manifests/emissary/emissary-crds.yaml
+COPY --from=emissary-builder /buildroot  /buildroot
+COPY --from=emissary-builder /newroot /
```

Comment:
lets do this as the first copy.

Reply:
Resolved in https://github.com/aws/eks-anywhere-build-tooling/pull/1059/commits/22874139be342205874c4a731894f9ad27c3e657